code/datums/traits/neutral.dm

```diff
@@ -0,0 +1,28 @@
+//traits with no real impact that can be taken freely
+//MAKE SURE THESE DO NOT MAJORLY IMPACT GAMEPLAY. those should be positive or negative traits.
+
+/datum/trait/no_taste
+	name = "Ageusia"
+	desc = "You can't taste anything! Toxic food will still poison you."
+	value = 0
+	mob_trait = TRAIT_AGEUSIA
```

Comment:
Not in scope, but this could be a nice minor brain trauma

Reply:
That is something that can happen to people when they have strokes.